docs/docs/api/withDecay.md

```diff
@@ -0,0 +1,75 @@
+---
+id: withDecay
+title: withDecay
+sidebar_label: withDecay
+---
+
+Starts a velocity based "scroll" animation.
+
+### Arguments
+
+#### `options` [object]
+
+Object containing animation configuration.
+Allowed parameters are listed below:
+
+| Options  | Default | Description                                  |
+| -------- | ------- | -------------------------------------------- |
+| velocity | 0       |                                              |
```

Comment:
I think we can also change the deceleration. Am I right?

Reply:
Yeah, you're right.